src/SfxWeb/src/app/views/partition/backups/backups.component.ts

```diff
@@ -23,12 +25,21 @@ export class BackupsComponent extends PartitionBaseControllerDirective {
 
   partitionBackupListSettings: ListSettings;
   actions: ActionCollection;
-
+  private debounceHandler: Subject<IOnDateChange> = new Subject<IOnDateChange>();
+  startDate: Date;
+  endDate: Date;
+  minDate: Date;
+  maxDate: Date;
+  startTime: any;
+  endTime: any;
+  private debouncerHandlerSubscription: Subscription;
   constructor(protected data: DataService, injector: Injector, private settings: SettingsService, public telemetry: TelemetryService) {
     super(data, injector);
   }
+  backupList: any;
+  dateRefresh: boolean;
```

Comment:
whats the reason for only refreshing once?

Reply:
thanks Jeffrey. I want the calendar to reset only when i refresh the page, instead of every time the list is refreshed. The refresh method seems to be called every seconds and hence was unable to filter out the list based on date and time

Reply:
Do you want it to function similar to how event store does tab does?

Reply:
yes. But the backup list is just a section of the backup tab. I wanted to set the date and time in "setup()" but the list data is not accessible there and I think from what i found is that ngOnInit() and setup() didn't work together